elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/ElideSecurityToPojo.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.parser;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.ElideHjsonUtil;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * The ElideSecurityToPojo class allows conversion of Security config Hjson 
+ * format to a Elide Security POJO.
+ */
+@Slf4j
+public class ElideSecurityToPojo {
```

Comment:
There is a lot of duplicate code between `ElideSecurityToPojo`, `ElideTableToPojo`, `ElideVariableToPojo`. I think we can have a single class like `ElideConfigToPojo` with all 3 methods taking 1 additional argument `String schemaType`. The parse method can then can do a `switch` statement  to call `ObjectMapper().readValue` with appropriate `.class` to transform to.